be/src/exec/json_scanner.cpp

```diff
@@ -343,6 +346,8 @@ Status JsonReader::_parse_json_doc(size_t* size, bool* eof) {
         return Status::OK();
     }
 
+    // clear memory here.
+    _origin_json_doc.GetAllocator().Clear();
```

Comment:
Why not just create a new `Document` each time we need to parse? It's not reasonable to make _origin_json_doc as a class member variable?

Reply:
It should work either. Just a pointer variable as a member.